_includes/header.html

```diff
@@ -1,8 +1,8 @@
-<header id="top" class="header">
+<!-- <header id="top" class="header">
```

Comment:
Instead of commenting this you should comment the section `_layouts/front.html` instead.

https://github.com/cvatch/cvatch.github.io/blob/master/_layouts/front.html